neo4j-inserter/insta_posts-inserter/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/codeuniversity/smag-mvp/kafka"
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	inserter "github.com/codeuniversity/smag-mvp/neo4j-inserter"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+)
+
+func main() {
+	neo4jHost := utils.GetStringFromEnvWithDefault("NEO4J_HOST", "localhost")
+	neo4jUsername := utils.GetStringFromEnvWithDefault("NEO4J_USERNAME", "neo4j")
+	neo4jPassword := utils.GetStringFromEnvWithDefault("NEO4J_PASSWORD", "12345678")
+
+	changesTopic := utils.GetStringFromEnvWithDefault("KAFKA_CHANGE_TOPIC", "postgres.public.posts")
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.GetStringFromEnvWithDefault("KAFKA_GROUPID", "neo4j-posts")
+	qReaderConfig := kafka.NewReaderConfig(kafkaAddress, groupID, changesTopic)
```

Comment:
FYI:
Maybe we can add a helper to DRY this up a little? (this quite similar in different inserter main files)